lib/oli/interop/ingest.ex

```diff
@@ -327,27 +327,11 @@ defmodule Oli.Interop.Ingest do
         {:error, :empty_project_title}
 
       title ->
-        labels =
```

Comment:
This code, which detects an object within the `children` array that has a `"type"` attribute equal to `"labels"` has to stay in place.  We have a tool (see https://github.com/Simon-Initiative/course-digest) that converts our legacy system courses into a "Torus Course Digest".  That tool places the label information from the Legacy courses into the hierarchy file.  So we need this to stay in place to read this type of digest structure. 

Maybe the easiest way to support this backwards compatibility would be to change the export of Torus projects to put the label information back into the hierarchy, as it is found here. 

The custom language settings can stay in the project file.

Reply:
Ok. I understand. I didn't think about legacy system courses when making these changes.